lib/galaxy/jobs/runners/cli.py

```diff
@@ -177,6 +180,9 @@ def __get_job_states(self):
         for job_destination_id, v in job_destinations.items():
             job_destination = v['job_destination']
             job_ids = v['job_ids']
+            if not job_ids:
```

Comment:
@mvdbeek By looking at how `job_destinations` is filled above, it seems that `job_ids` should never be empty at this point, am I missing something?